### PR TITLE
Added updateLocation mutation to the facade (frontend)

### PR DESCRIPTION
**Pull request by MrHardcode**
__Tasks__

* [x] Implement support for the geo-features the MapScreen needs in the facade
  * [x] `getNearbyUsers`
  * [x] `updatePosition`

### CODE REVIEW

File: FindYourFriends/screens/MapScreen.js [hunk 1]
```diff
@@ -12,8 +12,47 @@ import Card from "../components/Card";
...
+import useMutation from "@apollo/react-hooks";
 
 const MapScreen = (props) => {
+
+  const [updatePosition, { loading, error, data, called }] = useMutation(
+    facade.UPDATE_POSITION
+  );
+
+  // Example of how to use the hook
+  async function updateMyPosition(){
```
Comment (MalteMagnussen) [inline FindYourFriends/screens/MapScreen.js:66]:
```suggestion
  async function updateMyPosition(lon, lat){
```

File: FindYourFriends/screens/MapScreen.js [hunk 2]
```diff
@@ -12,8 +12,47 @@ import Card from "../components/Card";
...
+    await updatePosition({
+      variables: {
+        username: "name",
+        coordinates: {
+          lon: 12.57,
+          lat: 55.66
```
Comment (MalteMagnussen) [inline FindYourFriends/screens/MapScreen.js:72]:
```suggestion
          lon,
          lat
```

File: FindYourFriends/screens/MapScreen.js [hunk 3]
```diff
@@ -12,8 +12,47 @@ import Card from "../components/Card";
 import colors from "../constants/colors";
 import Input from "../components/Input";
 import facade from "../facade";
+import useMutation from "@apollo/react-hooks";
 
 const MapScreen = (props) => {
+
+  const [updatePosition, { loading, error, data, called }] = useMutation(
+    facade.UPDATE_POSITION
+  );
```
Comment (MalteMagnussen) [inline FindYourFriends/screens/MapScreen.js:21]: You sure this won't just fire immediately when the codes reaches here?

Comment (MrHardcode) [reply FindYourFriends/screens/MapScreen.js:21]: Why would it? It's not a `query`

File: FindYourFriends/screens/MapScreen.js (hunk 1 above)
Comment (MrHardcode) [reply FindYourFriends/screens/MapScreen.js:66]: I just made this code as an example - it's not meant to be used; it's meant to show how to use the hook

File: FindYourFriends/screens/MapScreen.js (hunk 2 above)
Comment (MrHardcode) [reply FindYourFriends/screens/MapScreen.js:72]: I may be **Mr. Hardcode** but I'm not *that* set on hardcoding things :P

File: FindYourFriends/screens/MapScreen.js (hunk 1 above)
Comment (MalteMagnussen) [reply FindYourFriends/screens/MapScreen.js:66]: Ahh okay.

File: FindYourFriends/screens/MapScreen.js (hunk 3 above)
Comment (MalteMagnussen) [reply FindYourFriends/screens/MapScreen.js:21]: Ok. I thought everything that didn't have "lazy" would fire immediately on Component Mount.

**Review by Castau — APPROVED** [review]

**Review by Runi-VN — APPROVED** [review]

**Pull request merged**